internal/topology/variables/clusterclass_variable_validation.go

```diff
@@ -132,8 +132,6 @@ func validateSchema(schema *clusterv1.JSONSchemaProps, variableName string, fldP
 	switch {
 	case len(apiExtensionsSchema.Type) == 0:
 		return field.ErrorList{field.Required(fldPath.Child("type"), "openAPIV3Schema.type cannot be empty")}
```

Comment:
As we don't want to point to nullable anymore, it's good enough if `type == "null"` is handled by the next case.